client-data/js/board.js

```diff
@@ -425,9 +425,22 @@ Tools.toolHooks = [
 			});
 		}
 
+		function wrapUnsetHover(f, toolName) {
+			// test for mobile device
+			// https://stackoverflow.com/a/24600597
+			if (/Mobi|Android/i.test(navigator.userAgent)) {
```

Comment:
If this is a screen size problem, then we should probably detect the screen size, not whether this is a mobile browser or not. User-Agent sniffing is generally not a good idea...